lib/dockerize_rails/docker_commands/helpers.rb

```diff
@@ -0,0 +1,27 @@
+module DockerizeRails
+  module DockerCommands
+    module Helpers
+      NAMES = {
+        image: {
+            rails: "#{DRConfig.application_name}_rails:#{DRConfig.application_env}".freeze,
+            mysql: "#{DRConfig.application_name}_mysql:#{DRConfig.application_env}".freeze,
+            postgres: "#{DRConfig.application_name}_postgres:#{DRConfig.application_env}".freeze,
+        }.freeze,
+        container: {
+            rails: "#{DRConfig.application_name}_rails_container".freeze,
+            mysql: "#{DRConfig.application_name}_mysql_container".freeze,
+            postgres: "#{DRConfig.application_name}_postgres_container".freeze,
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)

<!-- Comment by [SideCI](https://sideci.com) -->